osu.Desktop.VisualTests/KeyCounterTutorial/Count.cs

```diff
@@ -0,0 +1,123 @@
+﻿using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using OpenTK;
+using OpenTK.Graphics;
+
+namespace osu.Desktop.KeyCounterTutorial
+{
+    internal abstract class Count : Container
+    {
+        private int value;
+        public int FadeDuration => 200;
+        public bool IsCounting { get; set; }
+        public bool IsLit { get; set; }
+        public string Name { get; set; }
+
+        public int Value
+        {
+            get { return value; }
+            set
+            {
+                this.value = value;
+                ValueSprite.Text = value.ToString();
+            }
+        }
+
+        public Texture KeyUpTexture { get; set; }

```

Comment:
not sure about making these public when they are only used inside the class. we shouldn't be exposing more than we need to.